src/main/scala/sbtghactions/GenerativePlugin.scala

```diff
@@ -617,16 +617,6 @@ ${indentOnce(workflow.jobs.map(compileJob(_, sbt)).mkString("\n\n"))}
     } else {
       githubWorkflowSbtCommand.value
     }
-    //compileWorkflow(
```

Comment:
@987Nabil What is the context behind this test being commented, i.e. is it temporary to pass CI or is it intentional?

Reply:
It was only an example I copied for how the current call looks like. I forgot to remove it.

Reply:
I just realised it is already gone.